### PR TITLE
CUMULUS-1202 Fix bug in rules model

### PR DESCRIPTION
**Pull request by yjpa7145**

### CODE REVIEW

File: packages/api/tests/models/test-rules-model.js [hunk 1]
```diff
@@ -224,3 +225,51 @@ test.serial('it does not delete event source mappings if they exist for other ru
...
+    rule: {
+      type: 'kinesis',
+      value: randomString()
+    },
+    state: 'ENABLED'
+  });
+
+  const lambdaBefore = aws.lambda;
+  try {
+    aws.lambda = () => ({
```
Comment (markdboyd) [inline packages/api/tests/models/test-rules-model.js]: Is there a reason to not use `sinon` here?

Comment (yjpa7145) [reply packages/api/tests/models/test-rules-model.js]: This was quicker and, when [CUMULUS-1207](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1207) is addressed, this stub is going to be removed anyway.

Comment (markdboyd) [reply packages/api/tests/models/test-rules-model.js]: Fair enough, but we don't know when CUMULUS-1207 will get worked, and in the interim I'd prefer to be aligned with our stubbing approach in other tests. If for no other reason than consistency.

Would you mind updating to use `sinon.stub(aws, 'createEventSourceMapping').resolves({ UUID: randomString() });`? If it doesn't work for some reason, we don't need to worry about it

Comment (yjpa7145) [reply packages/api/tests/models/test-rules-model.js]: Changed in a6bac67ab1cc91f236fe9dfa244df1dc9c625ae7

**Pull request merged**